contracts/LineRegistry.sol

```diff
@@ -3,35 +3,23 @@
 pragma solidity ^0.8.17;
 
 import "@openzeppelin/contracts/access/Ownable2Step.sol";
-import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "./lines/base/LineMetadata.sol";
 
 contract LineRegistry is Ownable2Step {
-    using EnumerableSet for EnumerableSet.AddressSet;
-
-    // remoteChainId => localLineAddress[]
-    mapping(uint256 => EnumerableSet.AddressSet) private _localLineLookup;
+    // lineName => localLineAddress
+    mapping(string => address) private _localLineLookup;
```

Comment:
Could we rename _localLineLookup to _lineLookup, there is no `local` or `remote` concept.